### PR TITLE
fix(infiniteGrid): add defence code for cross-browser

### PR DESCRIPTION
**Pull request by sculove**
## Issue
#273

## Details
prevent for "Converting circular stucture to JSON" issue.
samsung browers(4.4.2/4.3) have only bugs.

## Preferred reviewers
@happyhj @kishu

### CODE REVIEW

File: src/infiniteGrid.js [hunk 1]
```diff
@@ -401,7 +401,9 @@ eg.module("infiniteGrid", ["jQuery", eg, window, document, "Outlayer"], function
 			var p;
 			for (p in this) {
 				if (this.hasOwnProperty(p) && /^_/.test(p)) {
-					data.push(p);
+					if (typeof this[p] !== "function" && !(this[p] instanceof HTMLElement)) {
```
Comment (jongmoon) [inline src/infiniteGrid.js]: How about ?

```js
if (this.hasOwnProperty(p) && /^_/.test(p)
   && typeof this[p] !== "function" && !(this[p] instanceof HTMLElement)) {
```

File: test/unit/js/infiniteGrid.test.js [hunk 2]
```diff
@@ -428,7 +428,9 @@ test("restore status", function(assert) {
 			var data=[];
 			for(var p in target) {
 			    if(target.hasOwnProperty(p) && /^_/.test(p)) {
+			    	if (typeof target[p] !== "function" && !(target[p] instanceof HTMLElement)) {
```
Comment (jongmoon) [inline test/unit/js/infiniteGrid.test.js]: `getProperties()` and related codes looks needless.

File: src/infiniteGrid.js (hunk 1 above)
Comment (sculove) [inline src/infiniteGrid.js]: ok

File: test/unit/js/infiniteGrid.test.js (hunk 2 above)
Comment (sculove) [inline test/unit/js/infiniteGrid.test.js]: good!

**Comment by jongmoon** [comment]
LGTM

**Comment by netil** [comment]
LGTM

**Pull request merged**